scripts/quick_start.sh

```
@@ -3,16 +3,21 @@
 echo "🎵 A Lo Cubano Boulder Fest - Quick Start"
 echo "========================================="
 
-cd "$(dirname "$0")"
+# Navigate to project root (parent of scripts directory)
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)